pages/k8s/upgrading.md

```diff
@@ -74,319 +74,24 @@ This may not include any customised configuration of Kubernetes, or user
-```
-
-Now add additional units of the kubernetes-worker. You should add as many units as you are replacing. For example, to add three additional units:
-
-```bash
-juju add-unit kubernetes-worker -n 3
-```
-
-This will create new units to migrate the existing workload to. As you configured the version prior to adding the units, they will be using the newly-selected version of **Kubernetes**.
-
-Now we can pause the existing workers, which will cause the workloads to migrate to the new units recently added. A worker unit is paused by running the corresponding action on that unit:
-
-```bash
-juju run-action kubernetes-worker/0 pause
-juju run-action kubernetes-worker/1 pause
-juju run-action kubernetes-worker/2 pause
-...
-```
-
-Continue until all the 'old' units have been paused. You can check on the workload status by running the command:
-
-```bash
-kubectl get pod -o wide
-```
-
-Once the workloads are running on the new units, it is safe to remove the old units:
-
-```bash
-juju remove-unit kubernetes-worker/0
-```
-
-Removing these units from the model will also release the underlying machines/instances they were running on, so no further clean up is required.
-
-<div class="p-notification--information is-inline">
-  <div markdown="1" class="p-notification__content">
-    <span class="p-notification__title">Note:</span>
-    <p class="p-notification__message">A variation on this method is to add, pause, remove  and recycle units one at a time. This reduces the resource overhead to a single extra instance.</p>
-  </div>
-</div>
-
-#### In-place upgrade
-
-To proceed with an in-place upgrade, first upgrade the charm itself:
-
-```bash
-juju upgrade-charm kubernetes-worker --channel=1.24/stable
-```
-
-Next, run the command to configure the workers for the version of **Kubernetes** you wish to run (as you did previously for the control-plane units). For example:
-
-```bash
-juju config kubernetes-worker channel=1.24/stable
-```
-
-All the units can now be upgraded by running the `upgrade` action on each one:
-
-```bash
-juju run-action kubernetes-worker/0 upgrade
-juju run-action kubernetes-worker/1 upgrade
-...
-```
-
-<a id='upgrading-series'> </a>
-
-## Upgrading the Machine's Series
-
-All of the charms support [upgrading the machine's series via Juju](https://juju.is/docs/upgrading-series).
-As each machine is upgraded, the applications on that machine will be stopped and the unit will
-go into a `blocked` status until the upgrade is complete. For the worker units, pods will be drained
-from the node and onto one of the other nodes at the start of the upgrade, and the node will be removed
-from the pool until the upgrade is complete.
+- [Upgrading from 1.20.x to 1.21.x](/kubernetes/docs/1.21/upgrading)
+- [Upgrading from 1.19.x to 1.20.x](/kubernetes/docs/1.19/upgrading)
+- [Upgrading from 1.18.x to 1.19.x](/kubernetes/docs/1.18/upgrading)
+- [Upgrading from 1.17.x to 1.28.x](/kubernetes/docs/1.17/upgrading)
```

Comment:
I think there are a few typos in here, see suggestion:

```suggestion
- [Upgrading from 1.20.x to 1.21.x](/kubernetes/docs/1.21/upgrading)
- [Upgrading from 1.19.x to 1.20.x](/kubernetes/docs/1.20/upgrading)
- [Upgrading from 1.18.x to 1.19.x](/kubernetes/docs/1.19/upgrading)
- [Upgrading from 1.17.x to 1.18.x](/kubernetes/docs/1.18/upgrading)
```

Reply:
Gosh, I made a mess of that!